WebExample/App.tsx

```diff
@@ -1,3 +1,3 @@
-import App from '../app';
+import App from '../app/src/examples/AnimatedStyleUpdateExample';
```

Comment:
For some reason React Navigation doesn't work properly with react-native-web  0.19 yet, here's the error:

<img width="419" alt="Zrzut ekranu 2023-05-15 o 18 12 37" src="https://github.com/software-mansion/react-native-reanimated/assets/20516055/9a3fca2c-e806-4771-9b61-305711b94290">

Reply:
We decided to downgrade react-native-web in WebExample to 0.18.10 for easier development. We'll upgrade to 0.19 once React Navigation supports it.